stats/conn.go

```diff
@@ -1,9 +1,10 @@
 package stats
 
 import (
-	"github.com/gorilla/websocket"
 	"net/http"
 	"strings"
+
+	"github.com/gorilla/websocket"
```

Comment:
When i save this file, GO auto-formats it to separate packages in GO from the external packages.